deployment/stack.py

```diff
@@ -147,10 +146,11 @@ def make_template(config, config_yaml):
         )
 
     volume_properties = {
-        'AvailabilityZone': config['AVAILABILITY_ZONE'],
         'Encrypted': True,
         'Size': config['DATA_VOLUME_SIZE'],
         'Tags': load_tags(),
+        'AvailabilityZone': functions.get_att(
+            'WebInstance', 'AvailabilityZone'),
```

Comment:
Technically, it's not a part of PEP8 but we decided this would increase readability. I'll update the coding style guide.